supervision/detection/line_zone.py

```diff
@@ -158,28 +158,28 @@ def trigger(self, detections: Detections) -> Tuple[np.ndarray, np.ndarray]:
             ]
         )
 
+        cross_products_1 = self._cross_product(all_anchors, self.limits[0])
+        cross_products_2 = self._cross_product(all_anchors, self.limits[1])
+        # anchor is in limits if it's on the same side of both limit vectors
+        in_limits = ~np.logical_xor(cross_products_1 > 0, cross_products_2 > 0)
+        # Reduce array to find out if all anchors for a detection are within limits
+        in_limits = np.min(in_limits, axis=0)
+
+        # Calculate which anchors lie to the left of the line
```

Comment:
Let's remove the comment; we're being fairly explicit that we care about one side later on.

Reply:
done